soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -101,6 +103,18 @@ Randomizer::Randomizer() {
         SpoilerfileCheckNameToEnum["Invalid Location"] = RC_UNKNOWN_CHECK;
         SpoilerfileCheckNameToEnum["Link's Pocket"] = RC_LINKS_POCKET;
     }
+
+    for (auto item: *RandoMain::GetFullItemTable()) {
+        // Easiest way to filter out all the empty values from the array, since we still technically want the 0/RG_NONE entry
+        if (item.GetName().english.empty()) continue;
```

Comment:
This is even better than my previous suggestion of checking for RG_NONE, since the one RG_NONE that is supposed to be there actually does have text associated with it.